bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -698,6 +706,35 @@ int handleDOMContentLoaded(long pView, long pArgs) {
 	return COM.S_OK;
 }
 
+int handleContextMenuRequested(long pView, long pArgs) {
+	ICoreWebView2ContextMenuRequestedEventArgs args = new ICoreWebView2ContextMenuRequestedEventArgs(pArgs);
+
+	long[] locationPointer = new long[1];
+	args.get_Location(locationPointer);
+	POINT pt = new POINT();
+	OS.MoveMemory(pt, locationPointer, POINT.sizeof);
+	pt.x = DPIUtil.autoScaleDown(pt.x); // To Points
+	pt.y = DPIUtil.autoScaleDown(pt.y); // To Points
+	Event event = new Event();
+	event.x = pt.x;
+	event.y = pt.y;
+	browser.notifyListeners(SWT.MenuDetect, event);
+	if (event.doit) {
```

Comment:
Currently, it behaves the opposite of how it is supposed to behave. I think this needs to be inverted (to also align with the IE implementation in `WebSite`):
```suggestion
	if (!event.doit) {
```

Reply:
oops